include/wx/animate.h

```diff
@@ -21,11 +21,19 @@
 #include "wx/bitmap.h"
 
 class WXDLLIMPEXP_FWD_CORE wxAnimation;
+class WXDLLIMPEXP_FWD_CORE wxGenericAnimation;
 
 extern WXDLLIMPEXP_DATA_CORE(wxAnimation) wxNullAnimation;
+extern WXDLLIMPEXP_DATA_CORE(wxGenericAnimation) wxNullGenericAnimation;
 extern WXDLLIMPEXP_DATA_CORE(const char) wxAnimationCtrlNameStr[];
 
 
+// ----------------------------------------------------------------------------
+// wxAnimation
+// ----------------------------------------------------------------------------
+
+WX_DECLARE_LIST_WITH_DECL(wxAnimationDecoder, wxAnimationDecoderList, class WXDLLIMPEXP_ADV);
+
```

Comment:
It doesn't matter much, but I don't see why did this have to be moved here. AFAICS it's only used in the generic header and could remain there. Couldn't it?

Reply:
I moved things around several times. That line was probably just left behind when things moved back.